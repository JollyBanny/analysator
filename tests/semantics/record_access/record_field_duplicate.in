var a: record
    x: integer;
    x: integer;
    end;

begin
end.