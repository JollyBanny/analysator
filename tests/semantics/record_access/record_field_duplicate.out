[Line: 3; Column: 5]	Duplicate identifier x
